omod/src/main/java/org/openmrs/module/adminui/page/controller/systemadmin/accounts/AccountPageController.java

```diff
@@ -109,9 +117,52 @@ public String post(PageModel model, @MethodParam("getAccount") @BindParams Accou
 	                   @SpringBean("adminService") AdministrationService administrationService,
 	                   @SpringBean("adminUiAccountValidator") AdminUiAccountValidator accountValidator,
 	                   @SpringBean("providerManagementService") ProviderManagementService providerManagementService,
+					   @SpringBean("appFrameworkService") AppFrameworkService appFrameworkService,
 	                   HttpServletRequest request, UiUtils uu) throws IOException {
 		
 		Errors errors = new BeanPropertyBindingResult(account, "account");
+
+        List<Extension> customUserFragments = appFrameworkService.getExtensionsForCurrentUser("createUser.manageAccountUserFragments");
+        Map<String, String[]> parameterMap = request.getParameterMap();
+        for(Extension ext : customUserFragments) {
+            if (StringUtils.equals(ext.getExtensionParams().get("type").toString(), "userProperty")) {
+                String userPropertyName = ext.getExtensionParams().get("userPropertyName").toString();
+                String[] parameterValues = parameterMap.get(userPropertyName);
+                if (parameterValues != null && parameterValues.length > 0) {
+                    if (parameterValues.length > 1) {
+                        log.warn("Multiple userProperty for a single user type not supported, ignoring extra values");
+                    }
+                    String parameterValue = parameterValues[0];
+                    if (userPropertyName !=null && parameterValue != null) {
+                        user.setUserProperty(userPropertyName, parameterValue);
+                    }
+                }
+            }
+        }
+
+        List<Extension> customPersonFragments = appFrameworkService.getExtensionsForCurrentUser("createUser.manageAccountPersonFragments");
+        for(Extension ext : customPersonFragments) {
+            if (StringUtils.equals(ext.getExtensionParams().get("type").toString(), "personAttribute")) {
+                String formFiledName = ext.getExtensionParams().get("formFieldName").toString();
+                String personAttributeTypeUuid = ext.getExtensionParams().get("uuid").toString();
+                String[] parameterValues = parameterMap.get(formFiledName);
+                if (parameterValues != null && parameterValues.length > 0) {
+                    if (parameterValues.length > 1) {
+                        log.warn("Multiple values for a single person attribute type not supported, ignoring extra values");
```

Comment:
if we sent the exception, then it will break the registration workflow. It will happen when the user configured the dashboard with duplicate filed ids. In the patient registration also, We followed this way.